File: benchmark/results/python-multilang/run-004/a-mcp/output/ordering/services/order_repository.py
from typing import Optional
from ordering.aggregates.order import Order

"""OrderRepository service."""
class OrderRepository:
    def create(self, input: Order) -> Order:
        raise NotImplementedError('not implemented')
    def findById(self, id: str) -> Optional[Order]:
        raise NotImplementedError('not implemented')
    def list(self, limit: float) -> list[Order]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

File: benchmark/results/java-invocation-script/run-002/a-mcp/output/ordering/services/OrderRepository.java
package com.metaengine.demo.ordering.services;

import com.metaengine.demo.ordering.aggregates.Order;

import java.util.List;
import java.util.Optional;

// OrderRepository service.

public class OrderRepository {

    public Order create(Order input) {
        throw new UnsupportedOperationException("not implemented");
    }

    public Optional<Order> findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    public List<Order> list(double limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}
